### PR TITLE
Persist peer stats

### PR DESCRIPTION
**Pull request by artemirk**

### CODE REVIEW

File: internal/httpapi/internal.go [hunk 1]
```diff
@@ -244,7 +245,7 @@ func exportIdentifiers(identifiers *types.PeerIdentifiers) *commonAPI.Connection
 	return oIdentifiers
 }
 
-func exportPeer(peer types.PeerInfo) (adminAPI.Peer, error) {
+func exportPeer(peer types.PeerInfo, mgr *manager.Manager) (adminAPI.Peer, error) {
```
Comment (RandoMan70) [inline internal/httpapi/internal.go:248]: Let's make it as a method of TunnelAPI struct. It has reference to manager inside.

File: internal/httpapi/status.go [hunk 2]
```diff
@@ -16,16 +16,19 @@ import (
...
-			PeersActive1d:   &stats.PeersActiveLastDay,
-			TrafficUp:       &stats.Upstream,
-			TrafficDown:     &stats.Downstream,
+			RestartRequired:  flags.RestartRequired,
+			PeersTotal:       &stats.PeersTotal,
+			PeersConnected:   &stats.PeersWithTraffic,
+			PeersActive1h:    &stats.PeersActiveLastHour,
+			PeersActive1d:    &stats.PeersActiveLastDay,
+			TrafficUp:        &stats.Upstream,
+			TrafficUpSpeed:   &upSpeed,
```
Comment (RandoMan70) [inline internal/httpapi/status.go:29]: Let's group counters first, then speeds.

File: internal/manager/events.go [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+	"github.com/vpnhouse/tunnel/internal/eventlog"
+	"github.com/vpnhouse/tunnel/internal/runtime"
+	"github.com/vpnhouse/tunnel/internal/types"
+	"github.com/vpnhouse/tunnel/pkg/human"
+	"github.com/vpnhouse/tunnel/proto"
+	"go.uber.org/zap"
+)
+
+const (
+	DefaultSentEventInterval = time.Second * 60 // 1minute
```
Comment (RandoMan70) [inline internal/manager/events.go:17]: Quote obvious comment...

File: internal/httpapi/internal.go (hunk 1 above)
Comment (artemirk) [reply internal/httpapi/internal.go]: Refactored

File: internal/httpapi/status.go (hunk 2 above)
Comment (artemirk) [reply internal/httpapi/status.go:29]: Sorted down

File: internal/manager/events.go (hunk 3 above)
Comment (artemirk) [reply internal/manager/events.go]: Use time.Minute %)

**Review by RandoMan70 — APPROVED** [review]

**Pull request merged**